### PR TITLE
Awesome books project!

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, we built a basic website that allows users to add/remove books from a list.

### CODE REVIEW

**Review by sinansevgi — COMMENTED** [review]
Hi @denisdiaconu and @TSHEPO-CLOUD , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README file. Please also remove the node_modules folder and add files that are not related to the project to the .gitignore file


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+  </head>
+  <body>
+    <section id="list">
+      <h1>List</h1>
+      <ul id="storage_list"></ul>
+    </section>
+    <section id="add_book">
+      <input type="text" id="title" placeholder="title" required/>
+      <input type="text" id="author" placeholder="author" required/>
+      <button type="button" id="btn">Add</button>
```
Comment (vichuge) [inline index.html:18]: - [ ] When I click the add button on deployment it doesn't work, I kindly invite you to fix it. 🐶 
- Hint: I'm almost sure the problem is on the src script, check it out.
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

- After fixing the button, the next CR could be able to test all other points of the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
Hello!
We fixed all the last code reviewer requesets.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] Update eslintrc.json to reflect the correct linting rules.
- [ ] Avoid disabling the entire eslint check for a section.


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  window.location.reload();
+};
+/* eslint-disable */ 
+const remove = (id) => {
+  const books = JSON.parse(localStorage.getItem('books'));
+  const newBooks = books.filter((e) => e.id.toString() !== id.toString());
+  localStorage.setItem('books', JSON.stringify(newBooks));
+  window.location.reload();
+};
+/* eslint-enable */
```
Comment (mikenath223) [inline src/js/index.js]: - [ ] Kindly avoid disabling the entire eslint rule. To improve the professionalism of your project consider fixing the linting issue or simply disable the specific rule.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "devDependencies": {
+    "@babel/core": "^7.14.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.0",
+    "@babel/preset-env": "^7.14.2",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^5.2.4",
+    "eslint": "^7.26.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.2",
```
Comment (mikenath223) [inline package.json:29]: - [ ] Ensure you use the same eslint config [rules](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) shared by other students. To update the rules consider raising a PR.

**Comment by denisdiaconu** [comment]
Update eslintrc.json to have the same config rules shared by other students.
Remove disabled eslint inside the js file.
Fix linters errors.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**